### DISCUSSION TITLE
Recommended way of upgrading to new OpenMDX releases

### DISCUSSION THREAD

**Discussion by javac9** [openmdx/openmdx#44] [Q&A] [answered]
The other day I successfully imported the project into Eclipse (OpenMDX v2.17.8). Now I can see there is a new version.

What is the recommended way of bringing the projects up-to-date, version-wise?

**Comment by wfro64** [comment, marked as answer]
Pull an build latest:

```
git pull
git checkout master
gradle clean
gradle assemble
gradle eclipse
```
Pull and build openmdx-v2.17.9:

```
git pull
git checkout openmdx-v2.17.9
gradle clean
gradle assemble
gradle eclipse
```
